packages/bot/src/components/Icons.jsx

```diff
@@ -1,33 +1,34 @@
-import React        from 'react';
-import arrow1       from '../assets/icons/ic-arrow-1.svg';
-import arrow2       from '../assets/icons/ic-arrow-2.svg';
-import logo         from '../assets/icons/ic-logo.svg';
-import tradeActive  from '../assets/icons/ic-trade-active.svg';
-import stage1       from '../assets/icons/ic-stage-1.svg';
-import stage2       from '../assets/icons/ic-stage-2.svg';
-import stage3       from '../assets/icons/ic-stage-3.svg';
-import stage4       from '../assets/icons/ic-stage-4.svg';
-import blueInfo     from '../assets/icons/icon-info-blue.svg';
-
+import React           from 'react';
+import arrow1          from '../assets/icons/ic-arrow-1.svg';
+import arrow2          from '../assets/icons/ic-arrow-2.svg';
+import logo            from '../assets/icons/ic-logo.svg';
+import tradeActive     from '../assets/icons/ic-trade-active.svg';
+import stage1          from '../assets/icons/ic-stage-1.svg';
+import stage2          from '../assets/icons/ic-stage-2.svg';
+import stage3          from '../assets/icons/ic-stage-3.svg';
+import stage4          from '../assets/icons/ic-stage-4.svg';
+import blueInfo        from '../assets/icons/icon-info-blue.svg';
+import iconInfoOutline from '../assets/icons/icon-info-outline.svg';
 /* eslint-disable react/display-name */
 export const Icon = svgItem => (props) => {
     const { className } = props;
     const vb = svgItem.viewBox.split(' ').slice(2);
 
     return (
-        <svg width={vb[0]} height={vb[1]} className={`icon icon-${className}`}>
+        <svg width={vb[0]} height={vb[1]} className={`icon ${className}`}>
```

Comment:
Can we use classNames here so we don't get `undefined` classes?

```suggestion
        <svg width={vb[0]} height={vb[1]} className={classNames('icon', { className: !!className }) }>
```